core_backend/app/user_tools/routers.py

```diff
@@ -42,9 +43,14 @@ async def create_user(
             user=user,
             asession=asession,
         )
+        await request.app.state.redis.set(
+            f"remaining-calls:{user_db.username}",
+            encode_api_limit(user_db.api_daily_quota),
+        )
         return UserCreate(
             username=user_db.username,
             content_quota=user_db.content_quota,
+            api_daily_quota=user_db.api_daily_quota,
```

Comment:
Why do we copy the quota limit from the user that is creating it? Shouldn't we do it from config?

Also, see `authenticate_or_create_google_user` - we also create an user when someone logs in for the first time using their google email. That is the primary use case where we need limits.

Reply:
It's not the user that is creating it but the user that is newly created. I see how it is confusing since we are using the same variable name. I will update it.